clarify aggregation doc a bit (#367)
graphite-ng/carbon-relay-ng: docs/aggregation.md

Aggregation
-----------

Aggregation is used to reduce the volume of datapoints, across time, across series, or across both.
Data corresponding to the same "bucket" (see below) is aggregated using the configured aggregation function such as sum or average.
Aggregators can also be (ab)used for quantizing (normalizing timestamps) or renaming series (but better to use a rewriter instead).

## input

Any data that "matches" is routed into the aggregator.

> Matching can be done using a prefix, substring and regular expression. They are optional, but incoming data must all of the match options that are set.

Reviewer: ```suggestion
Matching can be done using a prefix, substring and regular expression. They are optional, but incoming data must match all of the options that are set.
```

> note: prefix and substring are a good way to reduce CPU usage, as it allows to skip regex matching for incoming data if they don't match the substring/prefix.

Reviewer: ```suggestion
note: prefix and substring matching is a good way to reduce CPU usage, as it allows skipping regex matching for incoming data if it doesn't match the prefix or substring.
```
